### PR TITLE
veusz: change source to pypi

### PR DESCRIPTION
**Pull request by Kiri2002**
## Involved packages

 - veusz

## Involved issue

change source to pipi from github for elegant and easy to auto-update

## Details

- [x] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [ ] Provide New Package
- [x] Fix the Packages
  - [x] PKGBUILD
  - [x] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

**Comment by Kiri2002** [comment]
@starsareintherose

**Comment by hubutui** [comment]
not sure why you wanna change source to pypi. you could also check github release or tag for auto-update.

**Comment by hubutui** [comment]
```yaml
pre_build_script: |
  update_pkgver_and_pkgrel(_G.newver.lstrip('veusz-'))
- source: github
  github: veusz/veusz
  use_latest_release: true
```
should work.

**Comment by Kiri2002** [comment]
> not sure why you wanna change source to pypi. you could also check github release or tag for auto-update.

Because I think the pypi usually more regular in form. So when one soft both in pypi and github, i will chose pypi.

File: BioArchLinux/veusz/lilac.yaml
[the remark this answers is not in the document]
Comment (starsareintherose) [reply BioArchLinux/veusz/lilac.yaml:1]: check it by yourslef

**Review by starsareintherose — CHANGES_REQUESTED** [review]
Follow all suggestions or it should be closed in three days

**Comment by Kiri2002** [comment]
> Follow all suggestions or it should be closed in three days

Which? Source in github/pypi or the 'syntax' in lilac?
for source, i believe pypi better, if i was wrong, please show me reason

for syntax, I don't where wrong, it just same as others in repo, and you don't want to tell me

**Pull request closed without merging**